manuscript/template.tex

```diff
@@ -0,0 +1,667 @@
+    \includegraphics[width=10.5 cm]{assets/datastructure.png}
+    \caption{Satellite Health Data Structure}
+	\label{fig:datastructure}
+    \end{figure}   
+\unskip
+
+\begin{figure}[H]
+    \includegraphics[width=10.5 cm]{assets/datastructure_file.png}
+    \caption{Data structure File Structure}
+	\label{fig:datastructure_file}
+    \end{figure}
+	\noindent   
+\unskip
+
+As shown in figure \ref{fig:datastructure}, the data structure, with YAML file named '1-SatelliteHealth.yaml' is for determining the satellite's health.  This data would be transmitted with the beacon data to be received by the ground station operator.  Note that this data structure is derived from the user stories 2 and 4.1 described in the prior section detailing user stories.  These user stories detail the ground station operator's tasks to view the satellite beacon data and to request satellite health data packet so that operator can verify that AODS sensors and GPS data are within nominal parameters.  Table \ref{tab:datastructure} details the various fields that would be required in this beacon data packet to accomplish the aforementioned tasks.
+
+Figure \ref{fig:uml_datastructure} is a UML diagram of mapping of user stories to data structures generated from the YAML file shown in figure \ref{fig:datastructure}.  The generation of this diagram and the tables via the YAML files presented herein showcases the docs-as-code approach.  YAML files structured as a code are then converted into easily human readable documents for presentation.
+
+\begin{figure}[H]
+    \includegraphics[width=10.5 cm]{assets/uml_datastructure.png}
+    \caption{UML Diagram of Data Structures}
+	\label{fig:uml_datastructure}
+    \end{figure}   
+\unskip
+
+\subsection{Document Generation}
+As noted multiple times throughout this article, there have been a number of figures and tables generated from the YAML files placed within the SeaLion mission architecture GitHub repository.  Many of the figures are UML diagrams that are auto-generated artifacts rendered from the M30ML modeling language and formatted using the Liquid template language.  This is the link for the docs-as-code approach.  YAML code files are used to generate documents for information sharing between group members.  This means that any changes made to the SeaLion mission architecture model can immediately be used to generate new documents.  Whether it be diagrams, tables, or text, continuous updating is ensured that any changes affecting dependencies within the mission architecture are kept in sync.  A conference proceeding manuscript presented in AIAA SciTech 2023 was created purely by a docs-as-code format \cite{scitech_proceeding}.  The team used a LaTeX template to automatically format the manuscript to the conference guidelines and subsequently inject items such as the generated diagrams, tables, and references directly into the manuscript.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Component Implementation}
```

Comment:
Maybe change `Component Implementation` to `m30ml extensions`, mentioning DOF as a extended schema of m30ml for modeling a bill of materials (parts, tools, assembly instructions, & supporting material) for open-source hardware (OSHW)

Reply:
Resolved bbd10cadce0dc6665d7e685b218b57c669172dc5